src/interface/src/app/standalone/planning-areas/planning-areas.component.html

```diff
@@ -102,24 +102,13 @@ <h3>Home</h3>
       (click)="viewPlan(row, $event)"></tr>
   </table>
 
-  <!--Placeholder-->
-  <div class="pagination-placeholder">
-    <button
-      *ngFor="let number of [].constructor(pages$ | async); let i = index"
-      (click)="goToPage(i + 1)"
-      [disabled]="i + 1 === pageOptions.page">
-      {{ i + 1 }}
-    </button>
-
-    <div>
-      Page size :
-      <select (change)="changePageSize($event)" [value]="pageOptions.limit">
-        <option value="5">5</option>
-        <option value="12">12</option>
-        <option value="50">50</option>
-      </select>
-    </div>
-  </div>
+  <sg-paginator
+    *ngIf="pages$ | async"
```

Comment:
I added this ngIf bc the paginator has some math with the pageCount on ngInit, and without it it doesn't render them properly

<img width="327" alt="Screen Shot 2024-06-24 at 11 25 09" src="https://github.com/OurPlanscape/Planscape/assets/358892/ca76a128-8acc-488d-a4b9-a4f3c88b04a7">

Reply:
Ok. I thought I fixed that in the paginator, but I'll review again; we shouldn't have to do this.